spanner/benchmarks/benchmarks.go

```diff
@@ -152,5 +158,9 @@ func percentiles(percentile int, latencies []int64) any {
 }
 
 func generateUniqueID() int64 {
-	return rand.Int64N(totalRecords)
+	return rand.Int64N(totalRecords) + 1
+}
+
+func getRandomWaitTime(waitTime int64) time.Duration {
+	return time.Duration(rand.Int64N(waitTime-1) + 1)
```

Comment:
I think that this should be:

```suggestion
	return time.Duration(rand.Int64N(2 * waitTime-1) + 1)
```

that will ensure that when you specify for example 10ms as the wait time, then the average wait time will be around 10ms.